main.css

```diff
@@ -0,0 +1,199 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300..700&display=swap");
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: Poppins, sans-serif;
+  display: grid;
+  place-items: center;
+  height: 100vh;
+  background: #e6e6e6;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  max-width: 327px;
+  margin-block: 2rem;
+  gap: 2rem;
+}
+
+img {
+  max-width: 100%;
+  display: block;
+  object-fit: cover;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 327px;
+  overflow: hidden;
+  box-shadow: 0 0.1rem 1rem rgba(0, 0, 0, 0.1);
+  border-radius: 1em;
+  background: #dfe1e6;
+  background: linear-gradient(to right, #fff, #ece9e6);
+}
+
+.card__body {
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+
+.tag {
+  align-self: flex-start;
+  padding: 0.25em 0.75em;
+  border-radius: 1em;
```

Comment:
- [ ] Kindly make sure that your design match the Figma template design as much as possible (e.g. font, colors, and margins)
Your design
![Capture](https://user-images.githubusercontent.com/85095702/159098957-7ab25341-836e-49d4-accb-95e96192db20.PNG)
![Capture2](https://user-images.githubusercontent.com/85095702/159098976-3a29e384-b6ce-4489-aee5-88d50205e2b2.PNG)

Figma te
![figma](https://user-images.githubusercontent.com/85095702/159098996-b02553c7-315e-4d87-832b-bb0e0449c529.PNG)
mplate
![figma2](https://user-images.githubusercontent.com/85095702/159099021-280bad91-32c7-4503-85a9-167e86027d5c.PNG)